tests/core/nogo/custom/custom_test.go

```diff
@@ -455,7 +455,7 @@ func Test(t *testing.T) {
 			wantSuccess: false,
 			includes: []string{
 				// note the cross platform regex :)
-				`.*[\\/]cgo[\\/]examplepkg[\\/]pure_src_with_err_calling_native.go:.*function must not be named Foo \(foofuncname\)`,
+				`examplepkg[\\/]pure_src_with_err_calling_native.go:.*function must not be named Foo \(foofuncname\)`,
```

Comment:
I'm not entirely sure why I needed to make this change. Any ideas?

Reply:
The previous path with `cgo/` prefix was created from https://github.com/bazelbuild/rules_go/blob/86a7f025d2fc052ab6154603b481cb7e4419598f/go/tools/builders/cgo2.go#L68-L71

Could it be because of the new `cgoGoSrcsPath` is being used instead?

In some analyzers setup, folks would have a regex to specifically exclude cgo-generated source code from being scanned. I think if we cannot ensure that cgo-generated source are not being excluded by default, we should provide some consistent naming patterns for folks to exclude them downstream.

Reply:
I don't see how cgo-generated sources could be excluded by us: They may be needed by an analyzer to understand where symbols originating from cgo come from. If you see a way to send them to the nogo binary but still somehow exclude them from actual analysis, that would be a great way to resolve this.

Files generated by cgo either have a filename that starts with an underscore or contain the comment `// Code generated by cmd/cgo; DO NOT EDIT.`. As that is also true with regular Go, I would prefer folks to just depend on that to exclude these files if needed.